GooglePlayInstant/Editor/PlayInstantIl2cppBuilder.cs

```diff
@@ -0,0 +1,75 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor
+{
+    /// <summary>
+    /// A handler for building apks using quick deploy.
+    /// </summary>
+    public static class Il2cppBuilder
+    {
+        private const string SplashScenePath = "play-instant-loading-screen-scene.unity";
+        private static string _apkPathName = Path.GetFullPath("base.apk");
+
+        private static bool _projectIsUsingIl2Cpp =
```

Comment:
Is it necessary to cache this as a static field? Can you just make the actual call each time you need this?